clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/TransferWgsCramV1Metadata.scala

```diff
@@ -146,7 +146,51 @@ case class TransferWgsCramV1Metadata(
   }
 
   override def setSinglePath(destination: String): TransferWgsCramV1Metadata = {
-    throw new Exception(s"`setSinglePath` not implemented for $getClass")
+    if (pathsToMove.length != 1) {
```

Comment:
Do we need to implement this for this class? When do we think it will be used?

Reply:
One of the integration tests (checking if we can move with just a registered cram) hit it. I don't know if it'll ever happen in real life, but I'd rather have it implemented than leave a runtime bomb.

Reply:
Seems like a lot of code to satisfy a test which (as far as I can tell) is not testing something which will happen in production.

Reply:
Why are setSinglePath and moveAllInto separate ... uh ... branches?

Reply:
^ Since you prod it, I think there's a way to collapse them into one. I'll merge this as-is (since it's huge) and open a follow-up.